src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SqlMetaData.cs

```diff
@@ -97,80 +97,44 @@ public sealed partial class SqlMetaData
-            new SqlMetaData("text", SqlDbType.Text,
-                    UnlimitedMaxLength, 0, 0, 0, DefaultStringCompareOptions, false),                // SqlDbType.Text
-            new SqlMetaData("timestamp", SqlDbType.Timestamp,
-                    8, 0, 0, 0, SqlCompareOptions.None, false),                // SqlDbType.Timestamp
-            new SqlMetaData("tinyint", SqlDbType.TinyInt,
-                    1, 3, 0, 0, SqlCompareOptions.None, false),                // SqlDbType.TinyInt
-            new SqlMetaData("varbinary", SqlDbType.VarBinary,
-                    MaxBinaryLength, 0, 0, 0, SqlCompareOptions.None, false),                // SqlDbType.VarBinary
-            new SqlMetaData("varchar", SqlDbType.VarChar,
-                    MaxANSILength, 0, 0, 0, DefaultStringCompareOptions, false),                // SqlDbType.VarChar
-            new SqlMetaData("sql_variant", SqlDbType.Variant,
-                    8016, 0, 0, 0, SqlCompareOptions.None, false),            // SqlDbType.Variant
-            new SqlMetaData("nvarchar", SqlDbType.NVarChar,
-                    1, 0, 0, 0, DefaultStringCompareOptions, false),                // Placeholder for value 24
-            new SqlMetaData("xml", SqlDbType.Xml,
-                    UnlimitedMaxLength, 0, 0, 0, DefaultStringCompareOptions, true),                // SqlDbType.Xml
-            new SqlMetaData("nvarchar", SqlDbType.NVarChar,
-                    1, 0, 0, 0, DefaultStringCompareOptions, false),                // Placeholder for value 26
-            new SqlMetaData("nvarchar", SqlDbType.NVarChar,
-                    MaxUnicodeLength, 0, 0, 0, DefaultStringCompareOptions, false),                // Placeholder for value 27
-            new SqlMetaData("nvarchar", SqlDbType.NVarChar,
-                    MaxUnicodeLength, 0, 0, 0, DefaultStringCompareOptions, false),                // Placeholder for value 28
-            new SqlMetaData("udt", SqlDbType.Udt,
-                    0, 0, 0, 0, SqlCompareOptions.None, false),            // SqlDbType.Udt = 29
-            new SqlMetaData("table", SqlDbType.Structured,
-                    0, 0, 0, 0, SqlCompareOptions.None, false),                // SqlDbType.Structured
-            new SqlMetaData("date", SqlDbType.Date,
-                    3, 10,0, 0, SqlCompareOptions.None, false),                // SqlDbType.Date
-            new SqlMetaData("time", SqlDbType.Time,
-                    5, 0, 7, 0, SqlCompareOptions.None, false),                // SqlDbType.Time
-            new SqlMetaData("datetime2", SqlDbType.DateTime2,
-                    8, 0, 7, 0, SqlCompareOptions.None, false),                // SqlDbType.DateTime2
-            new SqlMetaData("datetimeoffset", SqlDbType.DateTimeOffset,
-                   10, 0, 7, 0, SqlCompareOptions.None, false),                // SqlDbType.DateTimeOffset
-            };
+        private static readonly SqlMetaData[] s_defaults =
```

Comment:
This block of changes was just because the `isPartialLength` argument is not being used (since it was deleted)